spec/features/participant_remove_spec.rb

```diff
@@ -0,0 +1,31 @@
+describe 'removing a participant' do
+  before :each do
+    create :participant
+  end
+  context 'with admin privilege' do
+    it 'removes a participant' do
+      when_current_user_is :admin
+      visit participants_url
+      click_on 'Remove'
+      expect(page).to have_text 'Participant has been removed'
```

Comment:
I think this (and the other expected texts) look better with the period at the end.